src/Security/OpenIdLoginAuthenticator.php

```diff
@@ -99,4 +101,16 @@ public function supportsRememberMe()
     {
         return false;
     }
+
+    private function getProvider(): array
```

Comment:
Why the code duplication. This exact same function is also defined in `LoginController`